config/jobs/kubernetes/wg-k8s-infra/canaries/sig-node-containerd.yaml

```diff
@@ -59,6 +59,35 @@ periodics:
     testgrid-dashboards: wg-k8s-infra-canaries
     testgrid-tab-name: containerd-build-1.4-canary
     description: "builds release/1.4 branch of upstream containerd"
+- name: ci-containerd-build-1-5-canary
+  interval: 30m
+  cluster: k8s-infra-prow-build
+  labels:
+    preset-service-account: "true"
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210721-2b77449-master
+        args:
+          - --repo=github.com/containerd/containerd=release/1.5
+          - --repo=github.com/containerd/cri=release/1.5
```

Comment:
containerd 1.5 does not use or have a release/1.5 branch (cri repo was folded into the main repo)

see `ci-containerd-build-canary` rather than `ci-containerd-build-1-4-canary`

Reply:
I did notice a `release/1.5` branch on containerd (https://github.com/containerd/containerd/tree/release/1.5). If it makes sense, I can just remove the cri repo.